FAQ.md

```diff
@@ -1,5 +1,20 @@
 The Sequelize Command Line Interface (CLI) Frequently Asked Question
 
+## How to install sequelize-cli globally?
+```
+npm install sequelize-cli -g
```

Comment:
Not CLI specific, I expect an average developer to know how to install a package globally. Remove this one